include/RE/H/HUDNotifications.h

```diff
@@ -12,29 +15,19 @@ namespace RE
 	public:
 		inline static constexpr auto RTTI = RTTI_HUDNotifications;
 
-		struct UnkData1
+		struct Notification
 		{
-			struct UnkData2
-			{
-				void*         unk00;  // 00
-				std::uint64_t unk08;  // 08
-				void*         unk10;  // 10
-				std::uint64_t unk18;  // 18
-			};
-			static_assert(sizeof(UnkData2) == 0x20);
-
-			const char*        unk00;  // 00
-			std::uint64_t      unk08;  // 08
-			const char*        unk10;  // 10
-			std::uint64_t      unk18;  // 18
-			BSFixedString      unk20;  // 20
-			BSTArray<UnkData2> unk28;  // 28
-			std::uint64_t      unk40;  // 40
-			std::uint64_t      unk48;  // 48
-			std::uint64_t      unk50;  // 50
-			std::uint64_t      unk58;  // 58
+			BSString                text;             // 00 - size 10
+			BSString                status;           // 10 - size 10
+			BSFixedString           sound;            // 20 - size 08
+			BSTArray<BSFixedString> objectives;       // 28 - size 18
+			std::uint32_t           type = 0;         // 40
+			std::uint8_t            pad[4];           // 44
+			TESQuest*               quest = nullptr;  // 48
+			TESWordOfPower*         word = nullptr;   // 50
+			std::uint32_t           time = 0;         // 58 - gameTime + iObjectivesWaitTime
```

Comment:
```suggestion
			std::uint32_t           type{ 0 };         // 40
			std::uint32_t            pad44{ 0 };           // 44
			TESQuest*               quest{ nullptr };  // 48
			TESWordOfPower*         word{ nullptr };   // 50
			std::uint32_t           time{ 0 };         // 58 - gameTime + iObjectivesWaitTime
```

Reply:
`pad44` initialized to 0 is mistake or intended?

Reply:
It is an intentional decision to zero initialize all members of the struct.